### ISSUE TITLE
Errors in ORM3/DBAL4

### ISSUE THREAD

**Issue by KDederichs** [zenstruck/foundry#567]
Not sure if you're already aware (I've not seen anything related to it) but updating to ORM 3 and DBAL 4 breaks Foundry:

```
OutOfRangeException: Unknown property "mappedBy" on class Doctrine\ORM\Mapping\ManyToOneAssociationMapping

/app/vendor/doctrine/orm/src/Mapping/AssociationMapping.php:243
/app/vendor/zenstruck/foundry/src/Factory.php:463
/app/vendor/zenstruck/foundry/src/Factory.php:411
/app/vendor/zenstruck/foundry/src/Factory.php:111
/app/vendor/zenstruck/foundry/src/FactoryCollection.php:76
/app/vendor/zenstruck/foundry/src/ModelFactory.php:41
/app/tests/Integration/Service/Foo/FooServiceTest.php:71
```

Maybe that's fixed in the 2.0 draft, but I thought I'd bring it up non the less :)

**Comment by kbond** [comment]
We'd like to have it working in 1.x too (https://github.com/zenstruck/foundry/pull/556)

**Comment by kbond** [comment]
Fixed in #569.
